interface/therapy_groups/index.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * interface/therapy_groups/index.php routing for therapy groups
+ *
+ * Contains the routing for therapy groups controllers.
+ *
+ * Copyright (C) 2016 Shachar&Amiel <shachar058@gmail.com> <amielboim@gmail.com>
```

Comment:
For the copyright lines, separate these out:
``````
 * Copyright (C) 2016 Shachar Zilbershlag <shachar058@gmail.com>
 * Copyright (C) 2016 Amiel Elboim <amielboim@gmail.com>
``````
Another option that some companies do is instead making the copyright to their company (ie. Matrix with common email address to matrix). This is entirely up to your group, though.